cmake/AddGHelpTarget.cmake

```diff
@@ -77,6 +93,13 @@ function (add_ghelp_target docname lang entities figures dtd_files)
 
 
     install(DIRECTORY "${BUILD_DIR}"
-        DESTINATION "share/gnome/help/${docname}")
+        DESTINATION "share/help/${lang}")
+
+    # Cleaning
+    # ADDITIONAL_CLEAN_FILES is introduced in cmake 3.15 but we can't use it.
```

Comment:
Please say instead "Can be replaced with ADDITIONAL_CLEAN_FILES once we can adopt Cmake 3.15."

Reply:
I read the ADDITIONAL_CLEAN_FILES document and found they are not enough functionality. So I will change this code.